### PR TITLE
Add CI tests for azdev commands (setup, style, test, linter)

### PR DESCRIPTION
**Pull request by tjprescott**
Adds CI tests for `azdev setup/test/style/linter` to help protect against regressions. cc/ @yugangw-msft

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -120,3 +120,107 @@ jobs:
...
+    displayName: 'Use Python 3.7'
+    inputs:
+      versionSpec: 3.7
+
+  - bash: |
+      #!/usr/bin/env bash
+      set -ev
+      . scripts/ci/install.sh
+      # verify azdev linter works
+      azdev linter redis
```
Comment (marstr) [inline azure-pipelines.yml]: Is redis the choice just because it's a relatively simple command_module?

Comment (tjprescott) [reply azure-pipelines.yml]: Absolutely.

File: scripts/ci/install.sh [hunk 2]
```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+set -ev
+
+echo "Install azdev into virtual environment"
+python -m venv env
+. env/bin/activate
+pip install -U pip setuptools wheel
+pip install $(find ${BUILD_ARTIFACTSTAGINGDIRECTORY}/pypi -name *.tar.gz)
```
Comment (marstr) [inline scripts/ci/install.sh]: I'd prefer that we not embed ADO specific variables into our scripts. Mind pulling this out into an argument? This line would become:

`pip install $(find $1/pypi -name *.tar.gz)` or alternatively, we could do the same pushd to repo root with `pushd` and `${BASH_SOURCE[0]}` that we do elsewhere.

File: azure-pipelines.yml (hunk 1 above)
Comment (marstr) [reply azure-pipelines.yml]: Love it, makes lots of sense to me :)

File: scripts/ci/install.sh (hunk 2 above)
Comment (marstr) [reply scripts/ci/install.sh]: Offline conversation with @tjprescott, we agreed to address this with a follow-up issue instead of a change here.

Comment (tjprescott) [reply scripts/ci/install.sh]: Here's the issue: #111

**Pull request merged**